templates/partials/contents.html

```diff
@@ -21,6 +22,18 @@ <h2>{{item.title}}</h2>
     <p>{{item.description}}</p>
   {% endif %}
 
+
+  {%- if item.guidance -%}
+    {% set contents = item.guidance.contents %}
+      {% call onsPanel({
+      "id": "item-guidance-" ~ guidance_id,
+      "classes": "ons-u-mb-m"
+      }) %}
+        {% set contents = item.guidance.contents %}
+        {% include 'partials/contents.html' %}
+      {% endcall %}
+  {% endif %}
```

Comment:
Ok I think all contents should be fixed because this doesn't currently fit the design for guidance panels. Also line 27 should removed because it is duplicated on line 32

Reply:
Removed the duplication.